src/cockpit/channels/stream.py

```diff
@@ -94,13 +94,21 @@ async def stream_process(self, args, options):
         env.update(options.get('env') or [])
 
         logger.debug('Spawning process args=%s', args)
-        process = await asyncio.create_subprocess_exec(
-            *args,
-            cwd=cwd,
-            env=env,
-            stdin=asyncio.subprocess.PIPE,
-            stdout=asyncio.subprocess.PIPE,
-            stderr=stderr)
+        try:
+            process = await asyncio.create_subprocess_exec(
+                *args,
+                cwd=cwd,
+                env=env,
+                stdin=asyncio.subprocess.PIPE,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=stderr)
+        except FileNotFoundError as error:
+            raise ChannelError('not-found') from error
+        except PermissionError as error:
+            raise ChannelError('access-denied') from error
+        except Exception as error:
```

Comment:
The only question I have: I wonder if this should rather be OSError.

Reply:
Right, I think "yes".